src/core/core.js

```diff
@@ -77,7 +77,7 @@ module.exports = function() {
 			onClick: null,
 			defaultColor: 'rgba(0,0,0,0.1)',
 			defaultFontColor: '#666',
-			defaultFontFamily: "'Helvetica Neue', 'Helvetica', 'Arial', sans-serif",
+			defaultFontFamily: '\'Helvetica Neue\', \'Helvetica\', \'Arial\', sans-serif',
```

Comment:
Very minor, but I think I prefer the previous version which is easier to read and less character. What do you think? should we only allow double quotes to avoid escaping (`quotes: [2, single, { avoidEscape: true }]`)?